### PR TITLE
Switch all pure math/vector methods to use the iD SDK.

### PR DESCRIPTION
**Pull request by Bonkles**
This is the first in a potential series of PRs to cut iD over to use the iD-SDK math library (https://www.npmjs.com/package/@id-sdk/math). 

The iD-SDK is an effort to reduce the amount of code that needs to be cloned/forked for downstream projects from iD to do their work effectively. 

This PR does the following: 
* Removes the geo/vector code and unit tests (they're now embedded within the SDK, so no need to keep them in iD's codebase
* Replaces those methods / imports with SDK referenced methods

### CODE REVIEW

**Comment by mmd-osm** [comment]
Assuming I need to log an issue with iD-SDK,  how would I do that? Right now GitHub tells me that’s limited to collaborators.

_An owner of this repository has limited the ability to open an issue to users that are collaborators on this repository._

**Comment by Bonkles** [comment]
> Assuming I need to log an issue with iD-SDK, how would I do that? Right now GitHub tells me that’s limited to collaborators.
> 
> _An owner of this repository has limited the ability to open an issue to users that are collaborators on this repository._

Good point! As this is the first time the code will be 'public' and in use, we should change that policy and allow folks to submit issues. 

UPDATE:  You should now have the option in the repo to submit issues.

File: test/spec/spec_helpers.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ mocha.setup({
 expect = chai.expect;
 
 window.d3 = iD.d3;   // Remove this if we can avoid exporting all of d3.js
+window.sdk = iD.sdk;
```
Comment (mbrzakovic) [inline test/spec/spec_helpers.js:50]: Was reading this issue https://github.com/openstreetmap/iD/issues/4379 and my understanding is that re-export of d3 is done due to d3.event having some internal states => doesn't mean that this should be a common practice for all dependencies.
Curious: Could we just include id-sdk scripts in test/index.html directly (via <script src... )?

**Review by mbrzakovic — COMMENTED** [review]
@Bonkles please resolve the code merge conflicts

**Pull request closed without merging**